src/gui_common/dialogs/CustomConfirmationDialog.tscn

```diff
@@ -3,11 +3,11 @@
 [ext_resource path="res://src/gui_common/dialogs/CustomConfirmationDialog.cs" type="Script" id=1]
 [ext_resource path="res://src/gui_common/thrive_theme.tres" type="Theme" id=2]
 [ext_resource path="res://src/gui_common/fonts/Lato-Bold-AlmostSmaller.tres" type="DynamicFont" id=3]
-[ext_resource path="res://src/gui_common/fonts/Lato-Regular-Smaller.tres" type="DynamicFont" id=4]
+[ext_resource path="res://src/gui_common/CustomRichTextLabel.tscn" type="PackedScene" id=4]
```

Comment:
Why does this now have a rich text label? There's no place that I'm aware of that would require rich text in a confirmation dialog.